CYD-Klipper/src/conf/global_config.cpp

```diff
@@ -131,4 +132,8 @@ void load_global_config()
     preferences.begin("global_config", true);
     preferences.getBytes("global_config", &global_config, sizeof(global_config));
     preferences.end();
+
+    temporary_config.debug = (bool) REPO_DEVELOPMENT;
```

Comment:
Also, this will go wrong when a release happens. REPO_DEVELOPMENT is undefined if it's a release build

Reply:
Done. 

```
    #if defined REPO_DEVELOPMENT  &&  REPO_DEVELOPMENT == 1
        temporary_config.debug = true;
    #else
        temporary_config.debug = false;
    #endif
```

Reply:
Somehow during search&replace a bunch of Serial.printf got swapped for LOG_LN (Serial.println wrapper) instead of LOG_F  while retaining the ((...)) syntax of LOG_F
Due to the ((...)) syntax, the inner paren was interpreted as comma expression, `("A=%d B=%d\n",A,B)` evaluating to `B` which Serial.println() was perfectly happy with.